check_sites.py

```diff
@@ -111,10 +115,14 @@ def main():
                 return
         old_checker = RwsCheck(old_sites, etlds, icanns)
         check_sets, subtracted_sets = find_diff_sets(old_checker.load_sets(), rws_checker.load_sets())
-        # TODO: add variable and check for subtracted_sets in case of user 
-        # removing old set from the list
     else:
         check_sets = rws_checker.load_sets()
+        if cli_primaries:
+            absent_primaries = [p for p in cli_primaries if p not in check_sets]
+            for p in absent_primaries:
+                error_texts.append("There was an error loading the set:\n" + 
+                    "could not find set with primary site " + p)  
```

Comment:
`p` is coming from the user, and we're looking for exact equality, so we ought to use quotes so that any whitespace or other mismatches are easier to spot.

```suggestion
                    f"could not find set with primary site \"{p}\"")  
```

Reply:
Done.